op-e2e/migration_test.go

```
@@ -0,0 +1,224 @@
+	zero := hexutil.Uint64(0)
+	one := hexutil.Uint64(1)
+	mptTimeOffset := hexutil.Uint64(60)
+
+	cfg := DefaultSystemConfig(t)
+	cfg.SetupMPTMigration = true
+	cfg.DeployConfig.L2GenesisDeltaTimeOffset = &zero
+	cfg.DeployConfig.L2GenesisEcotoneTimeOffset = &one
+	cfg.DeployConfig.L2GenesisKromaMPTTimeOffset = &mptTimeOffset
+
+	sys, err := cfg.Start(t)
+	defer sys.Close()
```

golangci-lint (1.62.2)

SA5001: should check error returned from cfg.Start() before deferring sys.Close()

(staticcheck)